### ISSUE TITLE
Console Errors upon importing Syphon a17 into Unity 5.2.2f1

### ISSUE THREAD

**Issue by jeanpoole** [Syphon/Unity3D#16]
Hello - 
importing a Syphon package ( Syphon for Unity Pro 4.0 1.0 Public Beta 2 a17) 
into Unity 5.5.2f1 personal (in a new, blank project… )

generates these error messages in my console: 

Assets/Plugins/Syphon.cs(106,28): warning CS0414: The private field `Syphon.updateContext' is assigned but its value is never used

and… 

WriteFile returned 25

which renders syphon unusable for me… 

Any ideas?

cheers!

jeanpoole@skynoise.net

**Comment by brianchasalow** [comment]
a couple things- 
1) that error is a non-issue. shouldn't cause problems.
2) I've never seen WriteFile returned 25.
3) 5.5.2f1 doesn't exist. what version are you actually running?

**Comment by jeanpoole** [comment]
Hi Brian - thanks for the reply - the problem I'm having is that Unity won't run / play - whenever there is *any* error message in my console... Apologies for the version mistake - I'm on 5.2.2f1... . Prior to my upgrade to that version, I was also having problems whenever error messages were in the console - however now I can run neither syphon or keijiro's funnel without getting error messages...  

RE: what error am i seeing? I cut and pasted the messages from my console - and yeah, not getting to display anything in syphon recorder at all, as unity won't run/play - it pops up a message instructing me to first fix errors mentioned in console...  

any thoughts on how to resolve this greatly appreciated..

thank

**Comment by jeanpoole** [comment]
Hi Brian - it seems my problem has been something to do with an external drive... and Unity not being able to properly write files to it for some reason.. When I ran a project from my laptop SSD - I was able to run the syphon plugin (with errors still in the console) and it sent out to my syphon recorder fine, even though the errors were displaying. It doesn't work when the project is on the external drive... no idea why... but I can make it work on my laptop now, so I'm happy. Thanks for the syphon plugin!
